ores/scoring/models/rev_id_scorer.py

```diff
@@ -62,6 +64,17 @@ def score(self, feature_values):
             }
         }
 
+    def quick_statistics(self):
+        '''Jam some data through to generate statistics'''
```

Comment:
The docstring is incorrect and violates PEP257, it should be "Jam some data through to generate statistics."
Also name of the method is better to be a verb.

Reply:
Good idea!